packages/components/search/src/search.js

```diff
@@ -0,0 +1,109 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper, helpers } from '@tradeshift/elements';
+import css from './search.css';
+
+import '@tradeshift/elements.icon';
+
+customElementDefineHelper(
+	'ts-search',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				autofocus: { type: Boolean, reflect: true },
+				dir: { type: String, reflect: true },
+				focused: { type: Boolean, reflect: true },
+				idleTime: { type: Number, attribute: 'idle-time', reflect: true },
+				placeholder: { type: String, reflect: true },
+				value: { type: String, reflect: true }
+			};
+		}
+
+		constructor() {
+			super();
+			this.autofocus = false;
```

Comment:
Is it necessary to define default value for them? Since we have the `reflect: true` it makes the DOM messy if you set default values!

Reply:
I just followed the docs about lit-element [static properties getter](https://lit-element.polymer-project.org/guide/properties#declare-properties-in-a-static-properties-getter):
> If you implement a static properties getter, initialize your property values in the element constructor.

I remove default values for boolean properties, because the element still working as expected.